upgrade/Upgrade_Only_CSM_with_iuf.md

```diff
@@ -34,10 +59,14 @@ using IUF.
 
 1. Management rollout
 
+   > **NOTE** The CSM services upgrade and validate CSM health are automatically executed in a hook run before the first management node is rolled out.
```

Comment:
Same comment on wording as in the other location.

Reply:
Fixed wording.